include/alpaka/acc/Tag.hpp

```diff
@@ -69,4 +70,21 @@ namespace alpaka
         alpaka::TagFpgaSyclIntel,
         alpaka::TagGpuSyclIntel>;
 
+    //!  \brief Function to print the names of each tag in the given tuple of acc tags
+    //!  \tparam TTuple is the type of the tuple of tags
+    template<typename TTuple>
+    void printAccTags()
```

Comment:
```suggestion
    void printTagNames()
```

Please avoid the term Acc where it is possible. This can easily cause confusion.